### ISSUE TITLE
Duplicate Duplicate 

### ISSUE THREAD

**Issue by earcam** [kelseyhightower/nocode#2695]
Duplicate of [#0](https://github.com/kelseyhightower/nocode/issues/0)

**Comment by earcam** [comment]
Possible duplicate of #2695

**Comment by NaniteFactory** [comment]
Yes, it's a duplicate. This problem is fixed on the current `master`.

**Comment by earcam** [comment]
ͦͦͦͦͦͦͦͦ ͚͚͚͚͚ͪ my bad.

Please close for now - I have a PR causing regression, we can reopen later

**Comment by earcam** [comment]
But regardless [#0](https://github.com/kelseyhightower/nocode/issues/2695) still needs addressing - given how long [#0](https://github.com/kelseyhightower/nocode/issues/2695)  has been open I think it's cavalier to ignore severity and continue adding features without addressing _core_ issues
